src/Sly/NotificationPusher/Adapter/Apns.php

```diff
@@ -87,8 +87,12 @@ public function push(PushInterface $push)
 
                 if (ServiceResponse::RESULT_OK === $response->getCode()) {
                     $pushedDevices->add($device);
-                }
-
+                }else{
+					$this->getOpenedClient()->close();
```

Comment:
Error! `getOpenedClient()` requires the $client param. Please check the method phpdoc.